src/style.css

```diff
@@ -0,0 +1,133 @@
+* {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+/* #region .card */
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-bottom: 17px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px;
+}
+
+.card__title-text {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  list-style-type: none;
+  padding: 8px 16px;
+}
+
+.card__title-text_small {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding-top: 4px;
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 16px;
+  padding-bottom: 0;
+}
+
+.card__price-title {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-coast {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button-inside {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  color: #ffff;
+  border-radius: 5px;
+  padding: 12px 56px 12px 55px;
+  text-decoration: none;
+  margin: 16px;
+}
+
+.card__button-inside:hover {
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  color: #00acdc;
+  background-color: #fff;
+}
+
+.card__feetback {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.card__responce {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: min-content;
+  padding-right: 16px;
+}
+/* #endregion */
+
+/* #region .stars */
+.stars {
+  box-sizing: border-box;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  padding-left: 16px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--4 {
+  display: flex;
+}
+
+.stars--4 > .stars__star:nth-child(-n+4) {
```

Comment:
all stars classes should work correctly how on hw with stars
stars--2 - 2 active stars